documentation/book/assembly-upgrade-cluster-operator.adoc

```diff
@@ -5,12 +5,8 @@
 [id='assembly-upgrade-cluster-operator-{context}']
 = Upgrading the Cluster Operator
 
-Supported paths to upgrade your Cluster Operator deployment to use a new version of {ProductName} are outlined in this section.
+The steps to upgrade your current Cluster Operator deployment to use {ProductName} {ProductVersion} are outlined in this section.
```

Comment:
Is it really _current Cluster Operator_ when there is a new one? Wouldn't it be better to say something like __older version of Cluster Operator_ or _previous version_?

Reply:
Well, it's their current deployment before they upgrade, but I've changed to 'older'